pkg/storegateway/bucket_stores_test.go

```diff
@@ -474,40 +474,63 @@ func TestBucketStore_Series_ShouldQueryBlockWithOutOfOrderChunks(t *testing.T) {
 
 	ctx := context.Background()
 	cfg := prepareStorageConfig(t)
+	fixtureDir := "fixtures/test-query-block-with-ooo-chunks"
```

Comment:
plz use `filepath.Join`

Reply:
Oh right. I forgot to use it here.

Reply:
Fixed in https://github.com/grafana/mimir/pull/7293/commits/75c2e3be3b080ef20b779bf2892dae5c93fc6f67